pkg/dashboards/dashboards.go

```diff
@@ -305,7 +305,12 @@ func createDashboard(dname string, orgid uint64) (map[string]string, error) {
 }
 func toggleFavorite(id string) (bool, error) {
 	// Load the dashboard JSON file
-	dashboardDetailsFname := config.GetDataPath() + "querynodes/" + config.GetHostID() + "/dashboards/details/" + id + ".json"
+	var dashboardDetailsFname string
+	if id == "10329b95-47a8-48df-8b1d-0a0a01ec6c42" || id == "a28f485c-4747-4024-bb6b-d230f101f852" || id == "bd74f11e-26c8-4827-bf65-c0b464e1f2a4" || id == "53cb3dde-fd78-4253-808c-18e4077ef0f1" {
```

Comment:
Can you make a function like `dashboardIsDefault(id)` that does this check, and then call that function here and on line 623 where you're doing a similar check? That way there's fewer opportunities for mistakes when checking if a dashboard is a default dashboard.